up.go

```diff
@@ -75,13 +75,32 @@ var (
 	unsafeMode   = pflag.Bool("unsafe-full-throttle", false, "enable mode in which command is executed immediately after any change")
 	outputScript = pflag.StringP("output-script", "o", "", "save the command to specified `file` if Ctrl-X is pressed (default: up<N>.sh)")
 	debugMode    = pflag.Bool("debug", false, "debug mode")
+	versionFlag = pflag.Bool("version", false, "display version and credits")
 )
 
 func main() {
 	// Handle command-line flags
 	pflag.Parse()
 
 	log.SetOutput(ioutil.Discard)
+	if *versionFlag {
+		os.Stdout.WriteString("up: Ultimate Plumber v" + version + " https://github.com/akavel/up\n")
+		os.Stdout.WriteString("up: \n")
+		os.Stdout.WriteString("up: Dependencies:\n")
+		os.Stdout.WriteString("up: \tTcell\n")
+		os.Stdout.WriteString("up: \t\thttps://github.com/gdamore/tcell\n")
+		os.Stdout.WriteString("up: \t\tLicense: Apache-2.0, ECL-2.0\n")
+		os.Stdout.WriteString("up: \n")
+		os.Stdout.WriteString("up: \tgo-isatty\n")
+		os.Stdout.WriteString("up: \t\thttps://github.com/mattn/go-isatty\n")
+		os.Stdout.WriteString("up: \t\tLicense: MIT, MIT-feh, JSON, Xnet\n")
+		os.Stdout.WriteString("up: \n")
+		os.Stdout.WriteString("up: \tpflag\n")
+		os.Stdout.WriteString("up: \t\thttps://github.com/spf13/pflag\n")
+		os.Stdout.WriteString("up: \t\tLicense: BSD-3-Clause, BSD-Source-Code, BSD-3-Clause-No-Nuclear-License-2014, BSD-4-Clause\n\n")
+		return

```

Comment:
Please change this to `os.Exit(0)`, to make it 100% clear what error code we're returning in this case, and to signal that we're doing this as a conscious decision.